dolphinscheduler-python/pydolphinscheduler/DEVELOP.md

```diff
@@ -83,6 +83,23 @@ black .
 flake8
 ```
 
+## Build Docs
+We use [sphinx][sphinx] to build docs. DS Python API CI would automatically build docs when you submit pull request in 
```

Comment:
We should use keyword `Dolphinscheduler` instead of short `DS`

```suggestion
We use [sphinx][sphinx] to build docs. Dolphinscheduler Python API CI would automatically build docs when you submit pull request in 
```

Reply:
Got it. Will fix it.